aptos-move/move-examples/mint_nft/step1/sources/nft_step1.move

```diff
@@ -0,0 +1,144 @@
+/// This module is the step 1 of our NFT Move tutorial. In this module, we go over how to create a collection, a token,
+/// and mint a token to a receiver.
+///
+/// How to interact with this module:
+/// 1. Create a source account.
+/// aptos init (this will create a default account)
+///
+/// 2. Publish the module.
+/// - 2.a Make sure you're in the right directory.
+/// We should run the following command in directory `aptos-core/aptos-move/move-examples/mint_nft/step1`.
+/// - 2.b Run the following CLI command to publish the module.
+/// aptos move publish --named-addresses mint_nft=[default account's address]
+/// (If you don't know the default account's address, run `nano ~/.aptos/config.yaml` to see all addresses.)
+///
+/// example output:
+    /*
+    step1 % aptos move publish --named-addresses mint_nft=a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde
+    Compiling, may take a little while to download git dependencies...
+    INCLUDING DEPENDENCY AptosFramework
+    INCLUDING DEPENDENCY AptosStdlib
+    INCLUDING DEPENDENCY AptosToken
+    INCLUDING DEPENDENCY MoveStdlib
+    BUILDING Examples
+    package size 2770 bytes
+    Do you want to submit a transaction for a range of [1164400 - 1746600] Octas at a gas unit price of 100 Octas? [yes/no] >
+    yes
+    {
+      "Result": {
+        "transaction_hash": "0x576a2e9481e71b629335b98ea75c87d124e1b435e843e7a2ef8938ae21bebfa3",
+        "gas_used": 11679,
+        "gas_unit_price": 100,
+        "sender": "a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde",
+        "sequence_number": 0,
+        "success": true,
+        "timestamp_us": 1669659103283876,
+        "version": 12735152,
+        "vm_status": "Executed successfully"
+      }
+    }
+    */
+/// - 2.c Check the module we just published on the Aptos Explorer.
+/// Go to https://explorer.aptoslabs.com/. For the example above, we used devnet, so we will choose `devnet` at the
+/// top right of the screen and paste the transaction_hash `0x576a2e9481e71b629335b98ea75c87d124e1b435e843e7a2ef8938ae21bebfa3`
```

Comment:
ah I see how it could be confusing; I used devnet so you'd need to run thru the steps yourselves and search for your unique transaction hash! I'll make it clearer here : )